scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
@spalger What is the benefit of that separation? Why add that level of indirection? I can understand the value of accessing things via `scripts` but why move things _outside_ of scripts? Like Court said, if the code isn't cohesive with anything in `src`, I wouldn't expect to find it there.